rules/feature_select.smk

```diff
@@ -0,0 +1,33 @@
+configfile: "configs/configuration.yaml"
+
+
+rule feature_select:
+    input:
+        expand("results/preprocessing/{plate_id}_normalized.csv.gz", plate_id=PLATE_IDS),
+    output:
+        expand(
+            "results/preprocessing/{plate_id}_feature_select.csv.gz",
+            plate_id=PLATE_IDS,
+        ),
+    params:
+        feature_select_config=config["config_paths"]["feature_select"],
+    conda:
+        "../envs/cytominer_env.yaml"
+    script:
+        "../scripts/feature_select.py"
+
+
+rule create_consensus:
+    input:
+        expand(
```

Comment:
**warning: beyond scope of this PR**

sometimes we want to create consensus profiles from more than just feature selected files.

We may also want to perform feature selection on all profiles (instead of just by plate).

How would this work? (we can discuss later too, but just something to keep in mind)

Reply:
Wrote it down on my notes ! I'll see what snakemake has to offer! 

Based on my current knowledge this, seems this can be done at the configuration level. We can provide a flag the types of consensus we want and the script can handle it.  

In addition, I feel that the consensus step  is general enough to create a modular function and call it when we are generating different consensus files.